go/multiorch/actions/scenario_detection.go

```diff
@@ -61,16 +61,16 @@ func (s InitializationScenario) String() string {
 // GatherInitializationStatus queries all nodes for their initialization status.
 // This is a helper function that multiorch can use to collect node status
 // before determining which initialization action to use.
-func GatherInitializationStatus(ctx context.Context, rpcClient rpcclient.MultiPoolerClient, cohort []*store.PoolerHealth, logger *slog.Logger) ([]*multipoolermanagerdatapb.InitializationStatusResponse, error) {
+func GatherInitializationStatus(ctx context.Context, rpcClient rpcclient.MultiPoolerClient, cohort []*multiorchdatapb.PoolerHealthState, logger *slog.Logger) ([]*multipoolermanagerdatapb.InitializationStatusResponse, error) {
 	statuses := make([]*multipoolermanagerdatapb.InitializationStatusResponse, len(cohort))
 
-	for i, node := range cohort {
+	for i, pooler := range cohort {
 		req := &multipoolermanagerdatapb.InitializationStatusRequest{}
-		status, err := rpcClient.InitializationStatus(ctx, node.ToMultiPooler(), req)
+		status, err := rpcClient.InitializationStatus(ctx, pooler.MultiPooler, req)
 		if err != nil {
 			if logger != nil {
 				logger.WarnContext(ctx, "Failed to get initialization status from node",
-					"node", node.ID.Name,
+					"node", pooler.MultiPooler.Id.Name,
```

Comment:
argh - I missed updating this one to node. We are planning to remove this one soon. So leaning towards leaving for now.